server/resolvers.js

```diff
@@ -7,14 +7,15 @@ const resolvers = {
     // get a single module by ID, for the module detail page
     getBreedsByName: (_, { breedName }, { dataSources }) => {
       return dataSources.catAPI.getBreedsByName(breedName);
-    },
+    }
   },
   Breed: {
     // get all images for a breed, given a breed id, also get 9 unique images
-    breedImage : ({ id }, _, { dataSources }) => {
-      return dataSources.catAPI.getImagesOfBreed(id, limit = 9);
-    },
-  },
+    breedImage: ({ id }, _, { dataSources }) => {
+      /*eslint no-undef: "off"*/
```

Comment:
Why?

Reply:
the 'limit' is not defined but still needed as an arg :/

Reply:
So what's expected to happen if limit is not defined?

Reply:
it's optional in the API anyway, without it, we'll have all the images for the breed returned ... with limi set, we are presetting max to be returned

Reply:
So why not just pass `undefined`/`null`/`-1` or whatever? ESLint is rightly emitting an error as such var is not defined.